--- x264-snapshot-20140328-2245/configure~	2014-03-29 08:43:39.000000000 +0100
+++ x264-snapshot-20140328-2245/configure	2014-03-29 08:44:32.177299477 +0100
@@ -950,7 +950,7 @@
 
 if [ "$gpac" = "auto" -a "$lsmash" != "yes" ] ; then
     gpac="no"
-    GPAC_LIBS="-lgpac_static"
+    GPAC_LIBS="-lgpac"
     cc_check "" -lz && GPAC_LIBS="$GPAC_LIBS -lz"
     if [ "$SYS" = "WINDOWS" ] ; then
         cc_check "" -lws2_32 && GPAC_LIBS="$GPAC_LIBS -lws2_32"
