--- libx264-0.1.2/common/i386/Makefile.am.orig	2004-12-17 13:27:36.000000000 +0100
+++ libx264-0.1.2/common/i386/Makefile.am	2005-10-11 22:33:13.052271512 +0200
@@ -3,7 +3,7 @@
 SUFFIXES = .c .o .lo .asm
 
 NASM=@NASM@
-NASMFLAGS=@NASMFLAGS@ $(CFLAGS)
+NASMFLAGS=@NASMFLAGS@
 
 noinst_LTLIBRARIES = lib_i386.la lib_nasm.la
 
@@ -19,8 +19,4 @@
 	$(NASM) $(NASMFLAGS) $< -o $@
 
 .asm.lo: $<
-	test -d .libs || mkdir .libs
-	$(NASM) $(NASMFLAGS) $< -o .libs/$*.lo
-	$(NASM) $(NASMFLAGS) $< -o $*.o
-	mv -f .libs/$*.lo $*.lo
-
+	$(LIBTOOL) --mode=compile --tag=NASM sh $(top_srcdir)/strip_fPIC.sh $(NASM) $(NASMFLAGS) $< -o $*.lo
--- libx264/strip_fPIC.sh.orig	Thu Jan  1 01:00:00 1970
+++ libx264/strip_fPIC.sh	Thu Feb 27 23:51:20 2003
@@ -0,0 +1,17 @@
+#!/bin/sh
+#
+# taken from flac sources  --qboosh
+#
+# libtool assumes that the compiler can handle the -fPIC flag
+# This isn't always true (for example, nasm can't handle it)
+command=""
+while [ $1 ]; do
+	if [ "$1" != "-fPIC" ]; then
+		if [ "$1" != "-DPIC" ]; then
+			command="$command $1"
+		fi
+	fi
+	shift
+done
+echo $command
+exec $command
