--- x264-snapshot-20140824-2245/configure~	2015-03-05 18:41:55.000000000 +0000
+++ x264-snapshot-20140824-2245/configure	2015-03-05 18:47:39.881867944 +0000
@@ -594,8 +594,8 @@
             if [[ "$asm" == auto && "$CFLAGS" != *-mfpmath* ]]; then
                 CFLAGS="$CFLAGS -mfpmath=sse -msse -msse2"
             fi
-            CFLAGS="-m32 $CFLAGS"
-            LDFLAGS="-m32 $LDFLAGS"
+            CFLAGS="$CFLAGS"
+            LDFLAGS="$LDFLAGS"
         fi
         if [ "$SYS" = MACOSX ]; then
             ASFLAGS="$ASFLAGS -f macho32 -DPREFIX"
@@ -627,7 +627,7 @@
         AS_EXT=".asm"
         ASFLAGS="$ASFLAGS -DARCH_X86_64=1 -I\$(SRCPATH)/common/x86/"
         stack_alignment=16
-        [ $compiler = GNU ] && CFLAGS="-m64 $CFLAGS" && LDFLAGS="-m64 $LDFLAGS"
+        [ $compiler = GNU ] && CFLAGS="$CFLAGS" && LDFLAGS="$LDFLAGS"
         if [ "$SYS" = MACOSX ]; then
             ASFLAGS="$ASFLAGS -f macho64 -DPREFIX"
             if cc_check '' "-arch x86_64"; then
@@ -641,7 +641,14 @@
                 RCFLAGS="--target=pe-x86-64 $RCFLAGS"
             fi
         else
+	  case "$host_os" in
+		*gnux32)
+			ASFLAGS="$ASFLAGS -f elfx32"
+		;;
+		*)
             ASFLAGS="$ASFLAGS -f elf64"
+	    ;;
+	  esac
         fi
         ;;
     powerpc*)
